docs/kubernetes.md

```diff
@@ -0,0 +1,71 @@
+# SOCI on Kubernetes
+
+This document explains how to configure SOCI on Kubernetes. For a hands on example, see [SOCI on Amazon Elastic Kubernetes Service (EKS)](./eks.md).
+
+> **Note**
+>
+> SOCI on Kubernetes has a few rough edges that you should consider before
+> using it for important workloads. See [Limitations](#limitations).  
+> We welcome your feedback and suggestions for improving the Kubernetes experience.
+
+## Configuration
+
+SOCI on kubernetes requires two pieces of configuration:
+1) [Containerd Configuration](#containerd-configuration) to launch containers with SOCI
+2) [Registry Authentication Configuration](#registry-authentication-configuration) so that SOCI can pull images from non-public container registries
```

Comment:
Was the wording "non-public" intentional?

Reply:
Yes. I was trying to distinguish between "anyone can access the image" (e.g. ECR public) and "anyone with credentials can access the image" (e.g. a private ECR repository with a permissive access policy). I'm not sure that distinction is really important to make though, so maybe this should just say private. What do you think?

Reply:
Would something a tad more verbose like `so that SOCI can pull images from registries requiring authentication` be a decent middle-ground?